ur_controllers/include/ur_controllers/passthrough_trajectory_controller.hpp

```diff
@@ -148,6 +162,9 @@ class PassthroughTrajectoryController : public controller_interface::ControllerI
   std::vector<control_msgs::msg::JointTolerance> goal_tolerance_;
   rclcpp::Duration goal_time_tolerance_ = rclcpp::Duration::from_nanoseconds(0);
 
+  realtime_tools::RealtimeBoxBestEffort<AsyncInfo> info_from_realtime_;
+  realtime_tools::RealtimeBoxBestEffort<AsyncInfo> info_to_realtime_;
```

Comment:
I think I don't quite understand why you want to use two real-time buffers. Wouldn't we want to use one for both directions?